### PR TITLE
VIH-9696 - Add an endpoint to a hearing at the last minute

### PR DESCRIPTION
**Pull request by twyfordr273**
### JIRA link (if applicable) ###

VIH-9696

### Change description ###

1. Created EndpointsUpdatedMessage model
2. Added getEndpointsUpdated function to event service which returns the endpoints updated observable
3. Created the showEndpointAdded function in the notification-toastr service
4. Added showEndpointAdded function to notification toastr service spy in waiting room based setup
5. Subscribed to handle the endpoints updated to handle the endpoints updated message in the waiting room base component
6. Added text to be displayed for notification and translations

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by oliver-scott — DISMISSED** [review]

**Review by phildebrah — DISMISSED** [review]
Tested and working successfully

File: charts/vh-video-web/values.dev.template.yaml [hunk 1]
```diff
@@ -9,3 +9,5 @@ java:
     AZUREAD__REDIRECTURI: https://${SERVICE_FQDN}/home
     EJUDAD__POSTLOGOUTREDIRECTURI: https://${SERVICE_FQDN}/logout
     EJUDAD__REDIRECTURI: https://${SERVICE_FQDN}/home
+    DOM1__POSTLOGOUTREDIRECTURI: https://${SERVICE_FQDN}/logout
```
Comment (oliver-scott) [inline charts/vh-video-web/values.dev.template.yaml:12]: @shaed-parkar should these changes be in here?

File: charts/vh-video-web/values.yaml [hunk 2]
```diff
@@ -21,6 +21,10 @@ java:
         - azuread--vhvideowebclientid
         - connectionstrings--signalr
         - connectionstrings--rediscache
+        - name: dom1--tenant--id
```
Comment (oliver-scott) [inline charts/vh-video-web/values.yaml:24]: @shaed-parkar should these changes be in here?

File: charts/vh-video-web/values.yaml [hunk 3]
```diff
@@ -77,6 +81,8 @@ java:
     AZUREAD__REDIRECTURI: https://vh-video-web.{{ .Values.global.environment }}.platform.hmcts.net/home
     EJUDAD__POSTLOGOUTREDIRECTURI: https://vh-video-web.{{ .Values.global.environment }}.platform.hmcts.net/logout
     EJUDAD__REDIRECTURI: https://vh-video-web.{{ .Values.global.environment }}.platform.hmcts.net/home
+    DOM1__POSTLOGOUTREDIRECTURI: https://vh-video-web.{{ .Values.global.environment }}.platform.hmcts.net/logout
```
Comment (oliver-scott) [inline charts/vh-video-web/values.yaml:84]: @shaed-parkar should these changes be in here?

File: charts/vh-video-web/values.dev.template.yaml (hunk 1 above)
Comment (shaed-parkar) [reply charts/vh-video-web/values.dev.template.yaml:12]: ah that's a mistake when I was working on the wrong branch

**Review by oliver-scott — APPROVED** [review]

**Review by shaed-parkar — APPROVED** [review]

**Review by phildebrah — APPROVED** [review]
QA approve

**Pull request merged**